lib/perso.ml

```diff
@@ -2187,6 +2193,46 @@ and eval_compound_var conf base env (a, _ as ep) loc =
     | _ -> raise Not_found
   end
 | sl -> eval_person_field_var conf base env ep loc sl
+and eval_title_field_var conf base env (_p, (cnt, name, title, places, dates)) _loc =
+    function
+    | ["is_first"] -> VVbool (begin match get_env "first" env with
+        Vbool x -> x
+      | _ -> raise Not_found
+      end)
+    | ["is_main"] -> VVbool (
+        match name with
+        | Tmain | Tnone -> true
+        | Tname _x -> false )
+    | ["cnt"] -> VVstring (string_of_int cnt)
+    | ["name"] ->
+        begin match name with
+        | Tname n -> VVstring (sou base n |> escape_html :> string)
+        | _ -> VVstring ""
+        end
+    | ["title"] -> VVstring (sou base title |> escape_html :> string)
+    | ["places"] -> VVstring (List.fold_left (fun acc pl -> acc ^
+                    (if acc <> "" then ", " else "") ^ 
+                    (sou base pl |> escape_html :> string)) "" places)
+    | ["dates"] -> VVstring (List.fold_left (fun acc (d1, d2) -> acc ^
+          (if acc <> "" then ", " else "") ^
+        begin match d1, d2 with
+        | Some (Dgreg (dmy1, Dgregorian)), Some (Dgreg (dmy2, Dgregorian)) -> 
+            (DateDisplay.string_of_dmy conf dmy1 :> string) ^ " - " ^ 
+            (DateDisplay.string_of_dmy conf dmy2 :> string)
+        | Some (Dgreg (dmy1, Dgregorian)), None ->
+             (DateDisplay.string_of_dmy conf dmy1 :> string ) ^ " - "
+        | None, Some (Dgreg (dmy2, Dgregorian)) -> 
+             " - " ^ (DateDisplay.string_of_dmy conf dmy2 :> string)
+        | Some (Dtext dt1), Some (Dtext dt2) ->
+             ((dt1 |> escape_html :> string) ^ " - " ^
+              (dt2 |> escape_html :> string))
+        | Some (Dtext dt1), None ->
+             ((dt1 |> escape_html :> string) ^ " - ")
+        | None, Some (Dtext dt2) ->
+             (" - " ^ (dt2 |> escape_html :> string))
+        | _ -> "? - ?"
```

Comment:
if we have `Some (Dgreg (dmy1, Djulian|Dfrench|Dhebrew))` shouldn't we try to print it by converting it to Dgregorian first?

Also for clarity I would try to match on one date at the time if possible.
maybe something like:
```
let date_opt_to_string =
   match d with
   | Some ( Dgreg (dmy ...)) ->  DateDisplay.string_of_dmy conf dmy :> string
   | .... -> ...
   | None -> ""
in
Format.sprintf "%s - %s" (date_opt_to_string d1) (date_opt_to_string d2)
```